.github/workflows/cypress.yml

```diff
@@ -0,0 +1,38 @@
+name: End-to-end tests
+on: [pull_request]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: app-web/node_modules
+          key: ${{ runner.OS }}-build-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-build-${{ env.cache-name }}-
+            ${{ runner.OS }}-build-
+            ${{ runner.OS }}-
+      - uses: bahmutov/npm-install@v1
+        with:
+          working-directory: app-web
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: patricksimonian/github-action@6dc0f8790dccace7167b12f9aaa0e28b7dc81c7c 
```

Comment:
is this token okay to be public? consider github secret?

Reply:
thats a commit id :)